extensions/amp-vimeo/0.1/test/test-amp-vimeo.js

```diff
@@ -74,5 +85,19 @@ describes.realWin(
       const iframe = vimeo.querySelector('iframe');
       expect(iframe.src).to.equal('https://player.vimeo.com/video/2323?dnt=1');
     });
+
+    it('should append muted=1 if video is autoplay', async () => {
+      const vimeo = await getVimeo('0123', true, false, true);
+      const iframe = vimeo.querySelector('iframe');
+      expect(iframe.src).to.equal(
+        'https://player.vimeo.com/video/0123?muted=1'
+      );
+    });
+
+    it('unlayoutCallback', async () => {
+      const vimeo = await getVimeo('0123', true, false, true);
+      vimeo.unlayoutCallback();
+      expect(vimeo.querySelector('iframe')).to.be.null;
+    });
```

Comment:
Do we need to check if any other resources are relieved when we unlayout?

Reply:
`unlayoutCallback` clears the iframe, so verifying that. It also calls the `unlistenFrame_` callback, but that is not trivial to test (AFAICT it requires something like a stub to return a spy, which overly complicates things).